public/docs/ts/latest/guide/index.jade

```diff
@@ -1,10 +1,501 @@
+  These are examples of Angular's [template syntax](template-syntax.html). 
+  We will grow accustomed to that syntax and may even learn to love it.
+  We'll begin to explain it in a moment.
+  
+  Before we do, focus attention on the last line. 
+  The `<hero-detail>` tag is a custom element representing the `HeroDetailComponent`.
+  
+  The `HeroDetailComponent` is a *different* component than the `HeroListComponent` we've been reviewing.
+  The `HeroDetailComponent` (code not shown) presents facts about a particular hero, the
+  hero that the user selects from the list presented by the the `HeroListComponent`.
+  The `HeroDetailComponent` is a **child** of the the `HeroListComponent`.
+
+figure
+  img(src="/resources/images/devguide/intro/component-tree.png" alt="Metadata" align="left" style="width:300px; margin-left:-40px;margin-right:10px" )
+:marked
+  Notice how `<hero-detail>` rests comfortably among the HTML elements we already know. 
+  We can mix ... and will mix ... our custom components with native HTML in the same layouts.
+  
+  And in this manner we can and will compose complex component trees to build out our richly featured application.
+<br clear="all">
+
+.l-main-section
+<a id="metadata"></a>
+:marked
+  ## Angular Metadata
+figure
+  img(src="/resources/images/devguide/intro/metadata.png" alt="Metadata" align="left" style="width:150px; margin-left:-40px;margin-right:10px" )
+:marked
+<p style="padding-top:10px">Metadata tells Angular how to process a class.</p>
+<br clear="all">
+:marked
+  [Looking back](#component-code) at the `HeroListComponent`, we see that it's just a class.
+  There is no evidence of a framework, no "Angular" in it at all.
+  
+  In fact, it really is *just a class*. It's not a component until we *tell Angular about it*.
+  
+  We tell Angular that `HeroListComponent` is a component by attaching **metadata** to the class.
+  
+  The easy way to attach metadata in TypeScript is with a **decorator**. 
+  Here's some metadata for `HeroListComponent`:
++makeExample('intro/ts/app/hero-list.component.ts', 'metadata')
+:marked
+  Here we see the `@Component` decorator which (no surprise) identifies the class
+  immediately below it as a Component class.
+  
+  A decorator is a function. Decorators often have a configuration parameter. 
+  The `@Component` decorator takes a required configuration object with the
+  information Angular needs to create and present the component and its view.
+  
+  Here we see a few of the possible `@Component` configuration options:
+  
+  * `selector` - a css selector that tells Angular to create and insert an instance of this component 
+  where it finds a `<hero-list>` tag in *parent* HTML. 
+  If the template of the application shell (a Component) contained
+<div style="margin-left:30px">
+code-example(language="html").
+    &lt;hero-list>&lt;/hero-list>
+</div>  
+:marked
+  >Angular would replace that tag with an instance of the `HeroListComponent` view.
```

Comment:
I'd propose to change or remove this sentence, since Angular 2 **does not** replace a tag with a component's template. Due to Shadow DOM (Emulation), a component always needs a host element, which is the one we're specifying with `selector`. AFAIK, there isn't even a way to replace it.

This is also why the `replace` property in the DDO in A1 is marked as deprecated.